src/seq/index.rs

```diff
@@ -427,8 +442,9 @@ where R: Rng + ?Sized {
     IndexVec::from(indices)
 }
 
-trait UInt: Copy + PartialOrd + Ord + PartialEq + Eq + SampleUniform
-    + core::hash::Hash + core::ops::AddAssign {
+trait UInt:
+    Copy + PartialOrd + Ord + PartialEq + Eq + SampleUniform + core::hash::Hash + core::ops::AddAssign
+{
```

Comment:
Importing `Hash` and `AddAssign` would help slightly

Reply:
Done.